engine/src/main/java/org/terasology/rendering/particles/DataMask.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2015 MovingBlocks
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.terasology.rendering.particles;
+
+/**
+ * Created by Linus on 7-3-2015.
+ */
+public enum DataMask {
```

Comment:
Can you think of a better name, describing the kind of data mask better? Maybe `ParticleDataMask` would be enough already...